recipes/musl/musl_1.1.15.oe

```diff
@@ -0,0 +1,61 @@
+inherit autotools
+
+SRC_URI = "https://www.musl-libc.org/releases/musl-${PV}.tar.gz"
+
+#EXTRA_OECONF += "--enable-shared"
+RECIPE_TYPES = "machine sdk"
+
+MACHINE[unexport]       = "1"
```

Comment:
Why are you doing this?  I cannot seem to find any references to MACHINE environment variable in musl.  I suspect this is a copy-paste legacy from glibc recipe.  Please fix.